### PR TITLE
added killing sprees

### PR DESCRIPTION
**Pull request by JSaurusRex**
Adds killing sprees
Disabled by default
sv_kspree_kills is 0 by default but set to anything else its enabled (5 is previous default)

## Checklist

- [X ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test (especially base/) or added coverage to integration test
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

**Comment by JSaurusRex** [comment]
I don't see any comments

**Comment by ChillerDragon** [comment]
> I don't see any comments

lmao my review was pending since days

github skill 0

File: src/engine/shared/variables_insta.h [hunk 1]
```diff
@@ -27,6 +27,7 @@ MACRO_CONFIG_INT(SvGrenadeAmmoRegenResetOnFire, sv_grenade_ammo_regen_reset_on_f
 MACRO_CONFIG_INT(SvSprayprotection, sv_sprayprotection, 0, 0, 1, CFGFLAG_SERVER, "Spray protection")
 MACRO_CONFIG_INT(SvOnlyHookKills, sv_only_hook_kills, 0, 0, 1, CFGFLAG_SERVER, "Only count kills when enemy is hooked")
 MACRO_CONFIG_INT(SvKillHook, sv_kill_hook, 0, 0, 1, CFGFLAG_SERVER, "Hook kills")
+MACRO_CONFIG_INT(SvKillingspreeKills, sv_kspree_kills, 0, 0, 20, CFGFLAG_SERVER, "How many kills are needed to be on a killing-spree")
```
[the remark this answers is not in the document]
Comment (JSaurusRex) [reply src/engine/shared/variables_insta.h:30]: Sure that works, current configs use this name but I think its fine to change it

**Comment by ChillerDragon** [comment]
Cool thanks! Could you squash the commits into one so we do not add any git history to the files we dont want to touch :)

I can cover the merge conflict then.

**Comment by ChillerDragon** [comment]
Nice thanks!

**Pull request merged**